model/src/main/proto/oppia_logger.proto

```diff
@@ -22,6 +23,11 @@ message EventLog {
   // The activity context of the event.
   Context context = 4;
 
+  // The profile to which this event corresponds, or empty if this event is not tied to a particular
```

Comment:
I'd suggest not going out of order (i.e. putting this after "7" instead of between "4" and "5"), it'd be harder for someone who adds more to the proto later to follow the sequential numbering and they might miss that "8" exists.

Reply:
This actually gets a bit tricky since it's often more beneficial to logically organize proto fields vs. numerically (as numeric just provides an indication of _when_ something was added, but not its contextual importance relative to other fields). I'd rather not make the suggested change since it'd be inconsistent with other fields (such as the ``oneof`` in the nested ``Context.activity_context below`` in this proto).

I scanned through protobuf best practices and didn't see anything on managing the next ID (other than a suggestion to try and minimize the number of fields in a proto). I've seen other teams that manage especially large protos use a comment to track the next ID, but this would require CI enforcement to make sure it actually stays up-to-date.

Since there's not an obvious solution and this proto isn't very large (meaning it's not hard to find the next index), I'm opting to leave this as-is. Please let me know if you have any follow-up thoughts.